src/app/report/edit-display/edit-display.component.ts

```diff
@@ -143,15 +159,25 @@ export class EditDisplayComponent {
   }
 
   getReportValues(checkpointId: string): any {
```

Comment:
Can this be report type?

Reply:
I've ignored setting types as im still waiting on #416 to implement them.

Reply:
Then you can wait a long time (╯°□°)╯︵ ┻━┻